tests/cypress/support/commands.js

```diff
@@ -56,4 +61,15 @@ Cypress.Commands.add("checkA11yOfPage", () => {
      */
     true // does not fail tests for ally violations
   );
+
+  // check accessibility using pa11y
+  cy.pa11y({
+    threshold: 10,
+    standard: "WCAG2AA",
+  });
+
+  // check accessibility using lighthouse
+  cy.lighthouse({
+    accessibility: 90,
```

Comment:
Same concept as above. The lighthouse plugin can check lots of other things, but I've got it set currently to only care about accessibility. Right now it's set to fail if the a11y check scores less than 90/100, but again, wanted to start a discussion. We may decide to have different limits for local checks vs. CI checks, too.

Reply:
90 seems like a good one to aim for right now